src/main/java/com/microsoft/azure/functions/worker/broker/JavaFunctionBroker.java

```diff
@@ -96,6 +97,11 @@ private void createInvocationChainFactory(FunctionDefinition functionDefinition,
 					this.sdkParameterAnalyzer.getRegistry()));
 		}
 
+		boolean otelEnabled = Boolean.parseBoolean(System.getenv("JAVA_ENABLE_OTEL"));
```

Comment:
NIT: can you rename this as `JAVA_ENABLE_OPENTELEMETRY` since this will be visible externally? 
Also is there a constants/app settings files where this can be added?

Reply:
I would like us to move away from requiring an environment variable to enable OTel. With the .NET worker, we deliberately designed it to rely entirely on the template for enabling OTel.